tigerpath/majors_and_certificates/majors/ANT.json

```diff
@@ -244,7 +234,10 @@
                         "ANT 405",
                         "ANT 442",
                         "ANT 446",
-                        "ANT 461"
+                        "ANT 461",

```

Comment:
https://anthropology.princeton.edu/undergraduate/courses/pre-approved-electives

"ANT 387(link is external) Anthropologies of Climate and Change (ENV 387)
LAS 390 / ANT 392(link is external) Multispecies Worlding and Global Health Politics
ANT 426(link is external) We Were Never Alone: Multispecies Ecologies in the Anthropocene"